.rubocop.yml

```diff
@@ -87,3 +88,6 @@ Metrics/PerceivedComplexity:
 
 Style/CaseEquality:
   Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
What do you all think about this cop?

Reply:
Possible, but you can Enable it and let "rubocop -a" automatically add the frozen string comments.

Reply:
It makes sense to add the comment. Frozen Strings are the default for newer ruby versions. Adding the comment enables this for some older versions.